ably/export_test.go

```diff
@@ -147,5 +147,5 @@ type Result = result
 func (c *RESTChannels) Len() int {
```

Comment:
Not necessarily as part of this PR, but can we unexport or remove this? It isn't part of the spec, and arguably it's not very useful.

Reply:
Removed